api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml

```diff
@@ -1944,6 +1946,28 @@ components:
       - Scala
       title: SparkApplicationType
       type: string
+    Shuffle:
+      description: External Shuffle storage configuration
```

Comment:
The description of the object and its fields should bring more explanation and guidance about their purpose, effect and how to configure them.
Default values should appear if relevant.

we have good examples in the same file : `serviceAccountName`, `namespace`, `availabilityZone`